vizro-core/src/vizro/static/css/accordion.css

```diff
@@ -42,35 +42,38 @@
   overflow-anchor: none;
   background-color: inherit;
   letter-spacing: -0.014px;
-  padding-left: 0;
+  padding-top: var(--spacing-04);
   padding-bottom: var(--spacing-04);
```

Comment:
Could you double-check the height of the accordion buttons: https://www.figma.com/file/Raf81HNv2pUdvJ8IvE74yL/QB-DS_(v1.0.0)?node-id=17904%3A112724&mode=dev

They seem to have a height of 40px in the figma designs?